File: Codigo/Obligatorio/codigo/ArenaGestorFront/src/app/models/Bands/BandResultLocationDto.ts
import { BandResultCountryDto } from "./BandResultCountryDto";

export class BandResultLocationDto {
    locationId: Number = 0;
    place: String = "";
    street: String = "";
    number: Number = 0;
    country: BandResultCountryDto = new BandResultCountryDto();
}

File: Codigo/Obligatorio/codigo/ArenaGestor/ArenaGestor.APIContracts/Band/BandResultLocationDto.cs
namespace ArenaGestor.APIContracts.Band
{
    public class BandResultLocationDto
    {

        public int LocationId { get; set; }
        public string Place { get; set; }
        public BandResultCountryDto Country { get; set; }
        public string Street { get; set; }
        public int Number { get; set; }
    }
}
